org.vergecurrency.verge-qt.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id": "org.vergecurrency.verge-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "19.08",
+  "command": "verge-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    "--socket=wayland",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "verge",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/verged",
+        "install -m 0744 -D -t /app/bin/internal bin/verge-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/verge-cli",
+        "install -m 0744 -D -t /app/bin/internal bin/test_verge"
+      ],
+      "post-install": ["/app/bin/internal/test_verge"],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/vergecurrency/verge/releases/download/v6.0.2/verge-6.0.2-x86_64-linux-gnu.tar.gz",
+          "sha256": "cdcb797d9bb11e9fe8062acd9ca46c5bafb02c4868f3c02fd417037584efd721"
+        }
+      ]
+    },
+    {
+      "name": "misc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dp -m 644 org.vergecurrency.verge-qt.desktop /app/share/applications/org.vergecurrency.verge-qt.desktop",
+        "install -Dp -m 644 verge.svg /app/share/icons/hicolor/scalable/apps/org.vergecurrency.verge-qt.svg",
+        "install -Dp -m 644 org.vergecurrency.verge-qt.appdata.xml /app/share/metainfo/org.vergecurrency.verge-qt.appdata.xml",
+        "bash install_wrappers.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "org.vergecurrency.verge-qt.desktop"
+        },
+        {
+          "type": "file",
+          "path": "org.vergecurrency.verge-qt.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html